src/rgw/rgw_rest_swift.cc

```diff
@@ -858,8 +859,10 @@ int RGWPutObj_ObjStore_SWIFT::update_slo_segment_size(rgw_slo_entry& entry) {
       return r;
     }
     bucket = bucket_info.bucket;
+    pbucket_info = &bucket_info;
```

Comment:
this is storing a pointer to `RGWBucketInfo bucket_info;` which goes out of scope on the next line

Reply:
OK, I fixed it in pr to octopus, https://github.com/ceph/ceph/pull/40296,